app/views/ui-components/v1/layouts/_footer.html.slim

```diff
@@ -1,9 +1,9 @@
 footer.footer-wrapper#footer-uic class=("#{"pre_prod_nav_color" if ENV['ENROLL_REVIEW_ENVIRONMENT'] == 'true'}")
   .container
-    .col-md-6
+    .col-md-3
```

Comment:
this might be a miscommunication for styling, but this button should be `position: fixed` and not be in regular flow with other elements

Reply:
Ok I need more specifics, I don't speak CSS.

Reply:
can we just remove all this extra stuff and see how the button looks with no additional styling?